lnbits/extensions/scrub/tasks.py

```diff
@@ -25,9 +25,9 @@ async def wait_for_paid_invoices():
         await on_invoice_paid(payment)
 
 
-async def on_invoice_paid(payment: Payment) -> None:
+async def on_invoice_paid(payment: Payment):
     # (avoid loops)
-    if payment.extra.get("tag") == "scrubed":
+    if payment.extra and payment.extra.get("tag") == "scrubed":
```

Comment:
I think this should read:

``` python
if payment.extra and payment.extra.get("tag") == "scrubed":
```

so the function bails out even when `payment.extra` is not set, no?